draft-packages/zen-off-canvas/KaizenDraft/ZenOffCanvas/ZenOffCanvas.tsx

```diff
@@ -17,135 +20,150 @@ type Props = {
   colorScheme?: ColorScheme
 }
 
-type State = {
-  visibleMenus: string[]
+type ToggleMobileNavEvent = {
+  type: "TOGGLE_MOBILE_NAV"
+  payload: {
+    id: string
+    state: "open" | "close"
```

Comment:
Nitpick but I wonder if it'd make more sense to name this `closed` (a state) instead of `close` (an action).

Reply:
Done!